apps/user-portal/src/pages/user-profile.tsx

```diff
@@ -0,0 +1,105 @@
+ */
+
+import * as React from "react";
+import {Container, Form, Grid, Header} from "semantic-ui-react";
+import {InnerPageLayout} from "../layouts";
+import {User} from "../components";
+import {getProfileInfo} from "../actions/profile";
+import {createEmptyProfile} from "../models/profile";
+
+export class UserProfilePage extends React.Component <any, any> {
+
+    constructor(props) {
+        super(props);
+        this.state = createEmptyProfile();
+    }
+
+    private setProfileDetails(response) {
+        this.setState({
+            displayName: response.displayName,
+            username: response.username,
+            emails: response.emails,
+            lastName: response.lastName,
+            phoneNumbers: response.phoneNumbers,
+            organisation: response.organisation,
+            roles: response.roles,
+            proUrl: response.proUrl,
+        });
+    }
+
+    public componentWillMount() {
+        getProfileInfo()
+            .then((response) => {
+                    this.setProfileDetails(response)
+                }
+            );
+    }
+
+    public render() {
+        return (
+            <InnerPageLayout pageTitle="Profile">
+                <Container>
+                    <Form>
+                        <div>
+                            <p>Manage information about you, your sub profiles and your account in general</p>
+                        </div>
+                        <br/><br/>
+                        <Grid divided='vertically'>
+                            <Grid.Row columns={2}>
+                                <Grid.Column width={3}>
+                                    <User/><br/>
+                                    <Header as='h3'>Personal Information</Header>
+                                </Grid.Column>
+                                <Grid.Column>
+                                    Name<br/>
+                                    {this.state.displayName + " " + this.state.lastName}<br/><br/>
+                                    Email<br/>
+                                    {this.state.emails[0]}<br/><br/>
+                                    Username<br/>
+                                    {this.state.username}<br/><br/>
+                                </Grid.Column>
+                            </Grid.Row>
+
+                            <Grid.Row columns={2}>
+                                <Grid.Column width={3}>
+                                    Telephone Number<br/>
+                                    +940112345678<br/><br/>
+                                    <Header as='h3'>Aliases</Header>
+                                </Grid.Column>
+                                <Grid.Column>
+                                    Organisation<br/>
+                                    {this.state.organisation}<br/><br/>
+                                </Grid.Column>
+                            </Grid.Row>
+                            <Grid.Row columns={2}>
```

Comment:
Let's remove this section temporally and re-add once we can get the actual content for it